### PR TITLE
Improve session list item accessibility

### PR DESCRIPTION
**Pull request by yukilabo**
## Overview (Required)
- Improvements for TalkBack
  - Remove unnecessary contentDescription from item view
  - Add stateDescription to favorite button

It's not the best, but it's easier to read.

## Screenshot
Notice the TalkBack reading at the bottom of the screenshot.
### Session item
Before | After
:--: | :--:
"isFavoritedtrue" | "[Session title] [Room name] [Minutes]"
 | 

### Favorite button
Before | After
:--: | :--:
"bookmark icon, ボタン" | "ON, favorite, ボタン"
 | 

### Before
https://user-images.githubusercontent.com/3139391/189697654-9f1ba2ca-ebd0-42be-a28e-376b02f60e76.mp4
### After
https://user-images.githubusercontent.com/3139391/189697705-ce403daf-49de-4ff7-bf01-3f095a78b168.mp4

### CODE REVIEW

**Comment by fumiya-kume** [comment]
Awesome work for a11y! 🎉 ✨ 
I believe that it's better for everyone, also How about attache the recorded video with using `Talkback` ? ⏺️ 
My idea helps for your PR will easy to review/ behavior check for us. 🙇‍♀️

**Comment by yukilabo** [comment]
Yes, I'm cheking now.

**Comment by yukilabo** [comment]
hmmm...

**Comment by takahirom** [comment]
Could you run ./gradlew spotlessKotlinApply?

**Comment by tarumzu** [comment]
@yukilabo @takahirom Sorry. CI is failing due to #397🙏

File: feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt [hunk 1]
```diff
@@ -63,7 +61,11 @@ fun SessionListItem(
             }
         }
         IconButton(
-            modifier = Modifier.testTag("favorite"),
+            modifier = Modifier
+                .testTag("favorite")
+                .semantics {
+                    stateDescription = if (isFavorited) "ON" else "OFF"
```
Comment (takahirom) [inline feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/SessionListItem.kt:67]: Looks great

**Review by takahirom — APPROVED** [review]
Thanks to you. DroidKaigi app is becoming cooler and cooler for all people.

**Pull request merged**